### PR TITLE
Add quiet option to disable console log output from validations

### LINKED ISSUE #57: add a quiet option (-q)
Hi,

After the vladiate 0.0.20, I could get the shell exitcode for success or not.

Now I would like to add a quiet option (-q) so that when I call vladiate, it silently validates, if not it will throw an error.

Best

### PR DESCRIPTION
**Pull request by maleix**
Fixes #57.

Adding "-q" or "--quiet" to the vladiate execution disables the console log generated by validations.

If used programatically, the feature can be enabled adding `disable_console_log=True` to the class initialization 

The overall result can still be captured by return values.

### CODE REVIEW

File: vladiate/main.py [hunk 1]
```diff
@@ -162,13 +172,19 @@ def load_vladfile(path):
...
+        ).validate()
+    )
 
 
 result_queue = Queue()
+arguments = parse_args()
 
 
 def main():
+    global arguments
```
Comment (di) [inline vladiate/main.py]: Why is this necessary? `arguments = parse_args()` is set on the line below.

File: vladiate/vlad.py [hunk 2]
```diff
@@ -24,6 +25,8 @@ def __init__(
         self.delimiter = delimiter or getattr(self, "delimiter", ",")
         self.line_count = 0
         self.ignore_missing_validators = ignore_missing_validators
+        if disable_console_log:
+            self.logger.disabled = True
```
Comment (di) [inline vladiate/vlad.py]: This can just be `self.logger.disabled = disable_console_log` with no `if` statement.

File: README.rst [hunk 3]
```diff
@@ -341,6 +341,10 @@ Running Vlads Programatically
       Whether to fail validation if there are fields in the file for which the
       `Vlad` does not have validators. Optional, defaults to `False`.
 
+  :``disable_console_log=False``:
```
Comment (di) [inline README.rst]: Thoughts about just calling this argument `quiet`? I think I'd prefer that.

**Comment by maleix** [comment]
Thanks for the feedback di!
I updated the pull request.

What makes me doubt is how to get the `arguments` variable from the `_vladiate` function in main.py.
Making arguments global or passing it as parameter don't seem clean approaches.

**Comment by di** [comment]
Thanks! Released in [`vladiate==0.0.22`](https://pypi.org/project/vladiate/0.0.22/).
